plugin/src/main/java/io/jenkins/plugins/analysis/core/restapi/ToolApi.java

```diff
@@ -28,12 +33,17 @@
      *         the URL to the latest results
      * @param size
      *         the number of warnings
+     * @param sizePerSeverity
+     *         the number of warnings, grouped by severity. Assumption is that
+     *         this is not null, and that it is properly populated with counts
+     *         that when summed, total the "size" parameter.
```

Comment:
Your assumption is correct.
```suggestion
     *         the number of warnings, grouped by severity 
```

Reply:
Given that, I have removed that section of the comment in 4566a73e2088af70276a83271107f5f3c31775ad, which I assumed was requested. If not, I can restore it.